server/docker-entrypoint.sh

```diff
@@ -11,5 +11,8 @@ yarn wait:db
 # apply migrations
 yarn migrate:up
 
+# create superuser
+yarn seed:superuser
```

Comment:
I don't think we should add that to the entrypoint since it will force us to provide the superuser password in the environment all the time (event if the superuser exists in pyrog db)